README.md

```diff
@@ -425,6 +425,19 @@ general we discourage its use for a number of reasons:
   which doesn't help.)
 
 
+## Using rspec-mocks with minitest 5.6+
+
+The latest versions of minitest also define an `expect` method. To use the
+`expect` method defined by rspec-mocks with minitest, load the following file
+**after** minitest is loaded:
+
+```ruby
+require "rspec/mocks/minitest_integration"
+```
+
+minitest style `expect` usage should continue to work as normal. 
```

Comment:
One of the build failures is due to excess whitespace on the end of this line (we enforce not having excess EOL-whitespace to prevent future unneeded git churn for whitespace cleanup).